lib/src/themes/expression/property_expression.dart

```diff
@@ -12,3 +13,56 @@ class GetPropertyExpression extends Expression {
   @override
   bool get isConstant => false;
 }
+
+/// A function that returns the output value of the stop equal to the
```

Comment:
Unless I'm mistaken, this looks copied from https://docs.mapbox.com/style-spec/reference/other/

We can't have any code or docs copied from anywhere. Please remove the docs here, and instead link to the spec.